pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/patch/Patch.java

```diff
@@ -160,8 +160,7 @@ public static boolean patch(@NonNull String id, @NonNull String user, @NonNull N
                 if (result == 1) {
                     logger.info("Updated {} value in table {}, record ID={}", nameValue.getName(), dbElement.getTableName(), id);
                 } else {
-                    throw new DsmInternalError(toErrorMessage(dbElement.getTableName(), dbElement.getColumnName(),
-                            nameValue.getValue(), dbElement.getPrimaryKey(), id) + ": " + result + " rows updated");
+                    logger.warn("Updated {} values in table {}, record ID={} , rows updated: {}", nameValue.getName(), dbElement.getTableName(), id, result);
```

Comment:
It would be helpful for log readers to understand why this is a warning.
Perhaps starting the message like this would provide that information: "Updated more than one {} values..."